core/tools.js

```diff
@@ -579,6 +579,90 @@
 			}, milliseconds || 0 );
 		},
 
+		/**
+		 * Throttles `input` events (or any `input` calls)
+		 * and triggers `output` not more often than once per `minInterval`.
+		 *
+		 * After each `input` scheduled `output` will be destroyed and replaced with
```

Comment:
TBO I'm not sure if we need information about discarding arguments in preceding calls since preceding call (there is only one previously scheduled call) is discarded. However, I think that the sentence: 
> If multiple calls occur within a single `minInterval` time, the latest `input` call with its arguments will be used to schedule next `output` call.

best corresponds to how this function works. Although I'm not sure if it's enough understandable.

Reply:
I think it works well 👍